applications/luci-app-speedtest/Makefile

```diff
@@ -0,0 +1,17 @@
+# This is free software, licensed under the Apache License, Version 2.0
+#
+# Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI for speedtest
+LUCI_DEPENDS:=+speedtestcpp
+LUCI_DESCRIPTION:=LuCI support for Speedtestcpp
+
+PKG_MAINTAINER:=Hilman Maulana <hilman0.0maulana@gmail.com>
+PKG_VERSION:=1.1
```

Comment:
Is there a specific reason why the initial version isn't 1.0?

Reply:
I was careless, I'll fix it